### ISSUE TITLE
Persistent Questie Tracker Header

### ISSUE THREAD

**Issue by Grizz2202** [Questie/Questie#3408]
The questie tracker header used to disappear when you would not be tracking any quests. At some point, the questie tracker header just stays there forever, whether you're tracking quests or not. 

I know I can go into the settings and manually select the "Enable Header" checkbox, but who wants to do this every time i track quests? I also dont need a stupid quest tracker up when I'm raiding (im not doing quests). 

Please either tell me what I'm doing wrong, or figure out how to make the header auto-hide again when I'm not tracking any quests. Thank you!!!

**Comment by BreakBB** [comment]
We will tackle this with a new option to change the current behaviour (always show the Tracker header) back to how it was before (only show the Tracker header if the user has any tracked quests).

**Comment by Grizz2202** [comment]
Thank you, this is exactly what I was hoping for!

**Comment by Dyaxler** [comment]
This currently works as intended. or has been fixed.

Current behavior: If no Achievements and/or Quests are being tracked then BOTH the tracker and the tracker header are hidden. It does not mimic a Minimized state. Not sure when and how this got fixed. I'm not finding any reference to this in any commits. It's possible that the code I recently submitted in a [Pull Request](https://github.com/Questie/Questie/pull/4664) resolved this issue. If that is the case, then keep an eye out for the next release...
